http://bugs.gentoo.org/show_bug.cgi?id=306255

Result of 'checking for pgsql/libpq-fe.h' is not used anywhere, just drop it.
--- configure.in.orig	2010-02-24 10:27:45.043267799 +0100
+++ configure.in	2010-02-24 10:28:09.016074329 +0100
@@ -1324,7 +1324,7 @@
 AC_CHECK_HEADERS(netinet/tcp.h stdlib.h sys/time.h sunmath.h sys/stat.h)
 AC_CHECK_HEADERS(poll.h)
 
-AC_CHECK_HEADERS(exception exception.h terminate.h openssl/ssl.h pgsql/libpq-fe.h)
+AC_CHECK_HEADERS(exception exception.h terminate.h openssl/ssl.h)
 
 # QT
 AC_CHECK_HEADERS(qapplication.h qsocketnotifier.h qlineedit.h)
--- configure.orig	2010-02-24 10:27:55.573241035 +0100
+++ configure	2010-02-24 10:28:19.102698424 +0100
@@ -6906,7 +6906,7 @@
 done
 
 
-for ac_hdr in exception exception.h terminate.h openssl/ssl.h pgsql/libpq-fe.h
+for ac_hdr in exception exception.h terminate.h openssl/ssl.h
 do
 ac_safe=`echo "$ac_hdr" | sed 'y%./+-%__p_%'`
 echo $ac_n "checking for $ac_hdr""... $ac_c" 1>&6
